### PR TITLE
Team ORA messaging about potential race conditions

### PR DESCRIPTION
**Pull request by nsprenkle**
[EDUCATOR-4851](https://openedx.atlassian.net/browse/EDUCATOR-4851): Team ORA messaging about potential race conditions

- Adds a banner to team ORAs warning that "Note: this is a team assignment. Make sure to coordinate with your teammates to avoid overwriting each other's work."
- Version bump to 2.5.5
- Dependency bump

team-concurrency-proposed-1

FYI: @edx/masters-devs

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -35,7 +35,7 @@ def load_requirements(*requirements_paths):
 
 setup(
     name='ora2',
-    version='2.5.3',
+    version='2.5.4',
```
Comment (iloveagent57) [inline setup.py]: Someone cut a 2.5.4 release recently, make yours 2.5.5

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "edx-ora2",
-  "version": "2.5.2",
+  "version": "2.5.4",
```
Comment (iloveagent57) [inline package.json]: Same thing here.  Latest releases: https://github.com/edx/edx-ora2/releases

**Comment by jansenk** [comment]
Would anyone oppose a suggestion to make it a yellow "warn" box rather than a red "error" box?

**Comment by nsprenkle** [comment]
> Would anyone oppose a suggestion to make it a yellow "warn" box rather than a red "error" box?

Gabe had suggested red instead of yellow but I'm not attached to the coloring either way...

**Pull request merged**